awslimitchecker/runner.py

```diff
@@ -358,6 +361,11 @@ def console_entry_point(self):
             ta_refresh_timeout=args.ta_refresh_timeout
         )
 
+        if args.skip_service is not None:
+            if args.service is not None:
+                raise SystemExit(0)
```

Comment:
I actually implemented the `add_mutually_exclusive_group()` style arg, but wound up without a firm belief that they were, in fact, mutually exclusive.

If someone wanted to list `--service A B C` and also `--skip-service C`, I can't say it's strictly wrong, or even confusing.

Happy to remove this block (364-366) if you agree.